components/signup/signup.tsx

```diff
@@ -455,16 +459,25 @@ const Signup = ({onCustomizeHeader}: SignupProps) => {
         }
     };
 
+    function sendFirstAdminUserTelemetryEvents(errorId: string) {
+        const sbr = getSbr();
+        if (!sbr) {
+            trackEvent('first_admin_sign_up_page', errorId, {serverId: DiagnosticId});
```

Comment:
@pvev, please follow the guidelines in https://docs.google.com/document/d/1CJeMFxnwUq8njKd3S3zxnfqrwUz8zNDsTmUAYqgy_uo/edit#heading=h.3g3h5oyxrijf wrt to the naming of these events.

Reply:
Not super sure about the javascript code here, but I think this will result in an event with `category` equal to `first_admin_sign_up_page` and `type` equal to `errorId`. The error ID looks like a property rather than an event type.

I have one question regarding `serverId`. IIRC it is added automatically in `user_id` field. Is that the case? If yes, then it doesn't need to be added in `serverId` property.

Reply:
> I have one question regarding serverId. IIRC it is added automatically in user_id field. Is that the case? If yes, then it doesn't need to be added in serverId property.

Correct, the server id is part of the info we send in all rudder events, we do not need to add as additional metadata.

Reply:
> Not super sure about the javascript code here, but I think this will result in an event with category equal to first_admin_sign_up_page and type equal to errorId. The error ID looks like a property rather than an event type.

+1, the category should be `signup` to be consistent with the other telemetry captured in this screen and the event type should be reflective of the action like: 

```suggestion
            trackEvent('signup', 'page_view_sign_up_page');
```

Reply:
done!